net/src/ipc/spawn.rs

```diff
@@ -57,39 +60,54 @@ pub fn ipc_spawn(
     // transport info (multiaddr) for any p2p interface bindings
     let re_p2p = regex::Regex::new("(?m)^#P2P-BINDING#:(.+)$")?;
 
-    // the child process is ready for connections
-    let re_ready = regex::Regex::new("#IPC-READY#")?;
+    // the child process is ready for ipc connections
+    let re_ipc_ready = regex::Regex::new("#IPC-READY#")?;
+
+    // the child process is ready for p2p connections
+    let re_p2p_ready = regex::Regex::new("#P2P-READY#")?;
 
     // we need to know when our child process is ready for IPC connections
     // it will run some startup algorithms, and then output some binding
     // info on stdout and finally a `#IPC-READY#` message.
     // collect the binding info, and proceed when `#IPC-READY#`
```

Comment:
This comment is just slightly incorrect but should be corrected I thin

Reply:
fixed